### PR TITLE
Node Version Update, Disabled JBrowse (temporarily) & New Version of Bento-tools with new UI. 

### PR DESCRIPTION
**Pull request by iksheth**

### CODE REVIEW

File: public/injectEnv.js [hunk 1]
```diff
@@ -18,6 +18,6 @@ window.injectedEnv = {
   // IDP Auth url
   REACT_APP_NIH_AUTH_URL: 'https://stsstg.nih.gov/auth/oauth/v2/authorize',
   // Client IDs for IDP
-  REACT_APP_GOOGLE_CLIENT_ID: 'Sample id',
-  REACT_APP_NIH_CLIENT_ID: 'Sample id',
-  };
+  REACT_APP_GOOGLE_CLIENT_ID: '196014713877-0d926jpdd691roubuc0kpu6r6ha9b9t5.apps.googleusercontent.com',
```
Comment (jonkiky) [inline public/injectEnv.js:21]: update needed here.

File: src/bento/navigationBarData.js [hunk 2]
```diff
@@ -77,8 +81,9 @@ export const navBarData = [
 ];
 
 export const navBarCartData = {
-  cartLabel: 'MY Files',
+  cartLabel: '',
```
Comment (jonkiky) [inline src/bento/navigationBarData.js:84]: Is it intended to be empty

File: src/pages/jbrowseDetail/jbrowseDetailView.js [hunk 3]
```diff
@@ -1,164 +1,164 @@
...
-JBrowseDetail.defaultProps = {
-  options: {
-    variants: false,
-    alignments: false,
-  },
-  bamFiles: [],
-};
-
-export default withStyles(defaultFooterStyles, { withTheme: true })(JBrowseDetail);
+// import PropTypes from 'prop-types';
```
Comment (jonkiky) [inline src/pages/jbrowseDetail/jbrowseDetailView.js:1]: I would recommend to remove unused the code. but not big issue for me, it is up to you.

File: src/bento/navigationBarData.js (hunk 2 above)
Comment (iksheth) [reply src/bento/navigationBarData.js:84]: Yes as per new design this is moving to right side of icon.

File: src/pages/jbrowseDetail/jbrowseDetailView.js (hunk 3 above)
Comment (iksheth) [reply src/pages/jbrowseDetail/jbrowseDetailView.js:1]: Done.

File: public/injectEnv.js (hunk 1 above)
Comment (iksheth) [reply public/injectEnv.js]: Done.

File: src/pages/jbrowseDetail/jbrowseDetailView.js (hunk 3 above)
Comment (iksheth) [reply src/pages/jbrowseDetail/jbrowseDetailView.js:1]: @jonkiky I agree removing un used code will be helpful. but in sooner future JBrowse is going to use again from different library due to which code is kept in code base for now.

**Review by jonkiky — APPROVED** [review]

**Pull request merged**